File: Java/Gate Two/TaskSeven.java
import java.util.Scanner;

public class TaskSeven{

	public static void main (String [] args){
	
		Scanner input = new Scanner (System.in);

		int sum = 0;

		for (int count = 1; count <= 5; count++){
			System.out.printf("Enter number %d: ", count);
			int number = input.nextInt();
			sum += number;
		}

		System.out.printf("The sum of the 5 numbers is %d%n", sum);
	}

}

File: JavaScript/Gate Two/TaskSeven.js
const prompt = require('prompt-sync')();
let sumOfNumbers = 0;

for (let count = 1; count <= 5; count++){

	let number = Number(prompt("Enter a number: "));

	sumOfNumbers += number;
}

console.log("The sum of the 5 numbers is %d", sumOfNumbers);